rotkehlchen/utils/version_check.py

```diff
@@ -13,14 +13,23 @@ class VersionCheckResult(NamedTuple):
     download_url: Optional[str] = None
 
 
-def check_if_version_up_to_date() -> VersionCheckResult:
+def get_current_version(check_for_updates: bool) -> VersionCheckResult:
+    """Get current version of rotki. If check_for_updates is set to true it also checks
+    if a new version is available.
+    """
+    our_version_str = get_system_spec()['rotkehlchen']
+    if check_for_updates:
+        return check_if_version_up_to_date(our_version_str)
+    return VersionCheckResult(our_version=our_version_str)
+
+
+def check_if_version_up_to_date(our_version_str: str) -> VersionCheckResult:
```

Comment:
Since this is no longer used anywhere you can just make it all into one function. There is no reason for this function to exist anymore. Combine them.